src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -1026,11 +1026,18 @@ export default class DefaultAudioVideoController
     return null;
   }
 
-  private async enforceBandwidthLimitationForSender(maxBitrateKbps: number): Promise<void> {
+  private async enforceBandwidthLimitationForSender(
+    maxBitrateKbps: number
+  ): Promise<void> {
     if (this.meetingSessionContext.browserBehavior.requiresUnifiedPlan()) {
-      await this.meetingSessionContext.transceiverController.setVideoSendingBitrateKbps(
-        maxBitrateKbps
-      );
+      const encodingParams = this.meetingSessionContext.videoUplinkBandwidthPolicy.chooseEncodingParameters();
```

Comment:
If an empty map means "just use the max bitrate", how does the policy say "don't change anything"? Should we pass the bitrate (and the `Sender`?) as optional arguments to `chooseEncodingParameters`, and make the default policies set the `maxBitrate` encodign parameter, so only old policies take the no-entries route?